### PR TITLE
Fixes #16926: fix shell autocompletion

### LINKED ISSUE #16926: bash-autocompletion does not recognize files and paths
If your command parameter is a local file or path, it would be nice to have the "standard" autocompletion.
Would be OK if this is just done when using `/` or `.` as the first char.

### What steps will reproduce the problem?

     yii db/import /dumps[TAB][TAB]

### What is the expected result?

     yii db/import /dumps/my-dump.sql

### What do you get instead?

     yii db/import /dumps

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.15.1
| PHP version      | 7.2
| Operating system | Debian (Docker)

### PR DESCRIPTION
**Pull request by np25071984**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #16926

### CODE REVIEW

**Review by samdark — APPROVED** [review]
Looks good. Please add a line for changelog.

**Comment by GHopperMSK** [comment]
Can somebody tell me what is going on here? What should I do to pass _Travis CI_ build successfully?

**Comment by rugabarbo** [comment]
> Can somebody tell me what is going on here? What should I do to pass _Travis CI_ build successfully?

You must add your name to end of changelog line: https://github.com/yiisoft/yii2/pull/16996/commits/2b5c53e0e736b37705478f9badb560484729d24c#diff-6ed25333dcd29100ab12768833283a45R7

See other lines for example.
Build breaks due to incorrect changelog line format.

File: framework/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Yii Framework 2 Change Log
 2.0.16 under development
 ------------------------
 
+- Bug #16926: Fix shell autocompletion
```
Comment (rugabarbo) [inline framework/CHANGELOG.md]: Add your name here at the end in brackets.

**Review by rugabarbo — CHANGES_REQUESTED** [review]

**Comment by samdark** [comment]
Merged. Thanks!

File: framework/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ Yii Framework 2 Change Log
 2.0.16 under development
 ------------------------
 
+- Bug #16926: Fix shell autocompletion (GHopperMSK)
```
Comment (rob006) [inline framework/CHANGELOG.md]: This actually does not make sense - completion script is not distributed with framework, so it should not be mentioned in changelog.

Comment (rob006) [reply framework/CHANGELOG.md]: @samdark Where? I cant find it at https://github.com/yiisoft/yii2-framework

Comment (rob006) [reply framework/CHANGELOG.md]: Just look at extracted commit from this PR: https://github.com/yiisoft/yii2-framework/commit/b1d822809acee9c3c64fe844de3fbcf8b4c185e1 - it is just entry in changelog, without affecting framework itself.

Comment (samdark) [reply framework/CHANGELOG.md]: Ah, subsplit. Yes, you are right.

Comment (samdark) [reply framework/CHANGELOG.md]: https://github.com/yiisoft/yii2/commit/a4b12b14cacb0d9ddac3b71015aaa047d1580b9a

**Pull request merged**